conda/environments/all_cuda-118_arch-x86_64.yaml

```diff
@@ -38,6 +38,7 @@ dependencies:
 - nbsphinx
 - nccl>=2.9.9
 - networkx>=2.5.1
+- networkx>=3.0
```

Comment:
`rapids-dependency-file-generator` is including the networkx version used by `cugraph-nx`, and the less-restrictive version independently used by other cugraph packages.  This did not seem to be a problem when testing the env files (`mamba` installed networkx version 3.1).